action.yml

```diff
@@ -6,45 +6,75 @@ inputs:
   github_token:
     description: 'GITHUB_TOKEN'
     required: true
-  level:
+  working_directory:
+    description: |
+      Directory to run the action on, from the repo root.
+      Default is . ( root of the repository)
+    default: '.'
+    required: false
+  log_level:
```

Comment:
Can you revert this and use level? For the consistency to other actions and it think "log" is not the right word for this flag as well.

Reply:
Done in https://github.com/reviewdog/action-tfsec/pull/15/commits/ed16cc4635e1ad2277c23c17f80f360023774b17